chia/util/augmented_chain.py

```diff
@@ -109,6 +107,7 @@ def height_to_block_record(self, height: uint32) -> BlockRecord:
             ret = self._get_block_record(header_hash)
             if ret is not None:
                 return ret
+            return self._underlying.block_record(header_hash)
```

Comment:
I suppose you can do this now that `_height_to_hash` grows indefinitely. But what's the upside of this?

Reply:
i have to do this, since we remove from extra_blocks once we validate and move to the underlying cache there might be a block in the augmented height_to_hash that was moved

Reply:
I there must be a way to move forward the "fork-point" of the `AugmentedBlockchain` though, isn't there? I suppose then `self._height_to_hash.get()` would return `None` and we would fall back onto the underlying blockchain.